### ISSUE TITLE
License classifiers have been superseded by license expressions

### ISSUE THREAD

**Issue by dahankzter** [pylint-dev/pylint#10292]
### Bug description

```python
The pylint installation as part of pre-commit fails with the provided error. Editing the pre-commit local cash in the way that the error suggests makes it work again.
```

### Configuration

```ini

```

### Command used

```shell
During installation
```

### Pylint output

```python
setuptools.errors.InvalidConfigError: License classifiers have been superseded by license expressions (see https://peps.python.org/pep-0639/). Please remove:

          License :: OSI Approved :: GNU General Public License v2 (GPLv2)
```

### Expected behavior

I expect it to be installed

### Pylint version

```shell
pylint 3.3.5
```

### Additional dependencies

```python

```

**Comment by Pierre-Sassoulas** [comment]
(Not exactly a duplicate but will be fixed by #10289 anyway )

**Comment by dahankzter** [comment]
Are you sure? The test fix was to remove that license declaration:

License :: OSI Approved :: GNU General Public License v2 (GPLv2)

Which is defined in the classifiers.
